packages/typespec-autorest/test/formats.test.ts

```diff
@@ -71,4 +71,48 @@ describe("typespec-autorest: format", () => {
     );
     expectDiagnosticEmpty(diagnostics);
   });
+
+  it("ensures certain scalars emit int32 and int64 formats", async () => {
```

Comment:
we already have tests for that in `primitive-types.test.ts`

Reply:
Perfect. Just wanted to make sure this behavior was codified since it underlies the logic of the rule, which doesn't live here.

Reply:
Test removed.